tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -987,6 +987,32 @@ def test_correct_redirect_as_2fa_user_invited_member(self):
 
         assert resp.redirect_chain == [("/auth/2fa/", 302)]
 
+    @with_feature("organizations:idp-automatic-migration")
+    def test_anonymous_user_with_automatic_migration(self):
+        AuthProvider.objects.create(organization=self.organization, provider="dummy")
+        resp = self.client.post(self.path, {"init": True})
+        assert resp.status_code == 200
+
+        path = reverse("sentry-auth-sso")
+
+        # Check that we don't call send_one_time_account_confirm_link with an AnonymousUser
```

Comment:
- what is the expected behavior of this call? is there anything we can assert further than a 200?
- why would we expect this to call `send_one_time_account_confirm_link`, is there something about this setup that I'm missing? (also why didn't the change cause `test_flow_as_anonymous` to fail previously)?

Reply:
it seems like we'd need to set a up a case where there's an existing authIdentity with the same email for this case, no?

Reply:
ah i see, they don't make it all the way into the case, before it would actually error while attempting to see if the user has a usable password or in the other cases.